aws/resource_aws_ram_resource_share_accepter_test.go

```diff
@@ -31,8 +33,8 @@ func TestAccAwsRamResourceShareAccepter_basic(t *testing.T) {
 				Config: testAccAwsRamResourceShareAccepterBasic(shareName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckAwsRamResourceShareAccepterExists(resourceName),
-					resource.TestMatchResourceAttr(resourceName, "share_arn", regexp.MustCompile(`^arn\:aws\:ram\:.*resource-share/.+$`)),
-					resource.TestMatchResourceAttr(resourceName, "invitation_arn", regexp.MustCompile(`^arn\:aws\:ram\:.*resource-share-invitation/.+$`)),
+					resource.TestCheckResourceAttrPair(resourceName, "share_arn", principalAssociationResourceName, "resource_share_arn"),
+					testAccMatchResourceAttrRegionalARN(resourceName, "invitation_arn", "ram", regexp.MustCompile("resource-share-invitation/.+$")),
```

Comment:
Blah, in this case the `invitation_arn` account ID is from the other AWS account:

```
    TestAccAwsRamResourceShareAccepter_basic: testing.go:683: Step 0 error: Check failed: Check 3/9 error: aws_ram_resource_share_accepter.test: Attribute 'invitation_arn' didn't match "arn:aws:ram:us-west-2:111111111111:resource-share-invitation/.+$", got "arn:aws:ram:us-west-2:222222222222:resource-share-invitation/2bce9ffd-c64d-4ec3-9105-92bd450f6545"
```

We do not have a covering helper function for that type of ARN -- maybe we can add one? I don't think the `invitation_arn` is exposed by the other resource.

Reply:
I added `testAccCheckResourceAttrRegionalARNAccountID ` in #13173 that should help with this